### ISSUE TITLE
Accessibility: Missing focus and contrast for <a> elements (links)

### ISSUE THREAD

**Issue by krallowski** [humhub/humhub#8247]
I report this now for the specific link on the login page, but I guess it is a general problem about links, as text-decoration is set to none here:
https://github.com/humhub/humhub/blob/54dcbf70b8f83c3e968ffae7b3dc128b856ff3f6/static/scss/_base.scss#L21

### What steps will reproduce the problem?
1. Go to the login page
2. Part 1: Check the contrast of the link with any tool
3. Part 2: tab through the input fields until you reach the "Forgot your password?" link

### What is the expected result?
It is visibly marked that the focus of the user is at the element accourding to: https://www.w3.org/TR/WCAG22/#focus-visible 

### What do you get instead?
Just a slightly different color. A border as indicator would be great.

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.18.3
| PHP version      | 
| Operating system |

**Comment by krallowski** [comment]
The same problem on the login page appears to the altcha.org symbol. I wanted to raise the issue diectly in their repo, but I am not sure which theme you use or if that makes any difference, so if you agree you could maybe raise an issue at their repo mentioning the theme you use :)

**Comment by krallowski** [comment]
The same issue appears for the avatars of the members of a space and in tables such as the user list in the admin settings.
